docs/images/snippets.md

```diff
@@ -126,7 +123,7 @@ fn PrintTotalArea(circles: Slice(Cpp.Circle)) {
 #include "circle.h"
 #include "geometry.carbon.h"
```

Comment:
It'd be good to consistently use `import` here too, at least in place of `#include <vector>`.

If you wanted to update the other includes, we don't have rules yet for how to map Carbon libraries to C++ module names, but `import Carbon:Geometry;` seems like a good guess to use as a placeholder. Or you could use `import "geometry.carbon.h";` for now.

Reply:
I have opted for the latter choice. I felt the former choice would be more appropriate if we wrote circle.h to export a module (say `export module circle;`), but then I didn't want to guess what the consuming Carbon code would look like (`import Cpp module circle`?)